tests/Facebook/InstantArticles/Transformer/WPTransformerTest.php

```diff
@@ -73,6 +73,7 @@ public function testTransformerLikeWPContent()
 
         $this->assertEquals($expected, $result);
         // there must be 3 warnings related to <img> inside <li> that is not supported by IA
+        // And 1 warning related to the getter
```

Comment:
Are you sure this comment should be added?
If there are 3 related to img + 1, the test assertion should be 4.
I guess this came from some dirty merge.